src/toolbars/AudioSetupToolBar.cpp

```diff
@@ -749,7 +749,8 @@ void AudioSetupToolBar::CommonMenuItemSteps(bool audioSettingsChosen)
          PrefsPanel::Factories factories;
          factories.push_back(PrefsPanel::PrefsNode(DevicePrefsFactory));
 
-         ViewDeviceSettingsDialog dialog(&GetProjectFrame(mProject), mProject, XO("Audio Settings:"), factories, 0);
+         ViewDeviceSettingsDialog dialog(
+            &GetProjectFrame(mProject), mProject, {}, factories, 0);
```

Comment:
No mistake here?

Reply:
If you mean the changed tittle - consider it the bug fix :-)